docs/05-concepts/06-database/01-connection.md

```diff
@@ -62,3 +70,59 @@ To remove the database and __delete__ all data add the `-v` flag.
 $ docker compose down -v
 ```
 
+
+## Connecting to a Postgresql Server Instance
+
+Connecting to a Postgresql Server Instance is similar to connecting to a Docker Image, but there are some prerequisites to take into account:
+
+- Making sure that the machine/server allows incoming traffic
+- Creating a Postgresql user and database
+
+You can connect to Postgresql server that you have installed on your local machine, a self-hosted one, Google Cloud SQL, AWS RDS or SQL Server, or any other 3rd party service.
+
+### Connecting to a Local Postgresql
+
+If you want to connect to a local Postgresql Server (with the default setup) then the `development.yaml` will work just fine. The only thing you need to do is change the password in the `passwords.yaml`.
```

Comment:
Suggestion: Add instructions around the specifying port.

The default port for Postgres is 5432; whenever I have a local instance of Postgres, it usually occupies that port. I think that needs to be changed as the default `development.yaml` specifies port 8090.

Reply:
I will endorse this point. I had trouble connecting serverpod to localhost: Postgres until I changed the port to 5432 in development.yaml to database: port: 5432. Since Postgres will connect to port 5432 by default, it is required to amend this accordingly. After this change, serverpod instance will start smoothly with messages regarding apiServer, inSightsServer and webServer listening on respective ports normally.